### PR TITLE
cloud: update release notes for 0920

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
cloud: update release notes for 0920

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by zhoubasten — APPROVED** [review]
LGTM for "introduce the cost based invoice" for self-service user

**Comment by IANTHEREAL** [comment]
LGTM

**Review by langyuezhang — APPROVED** [review]
LGTM

File: tidb-cloud/release-notes-2022.md [hunk 1]
```diff
@@ -8,6 +8,28 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+* Provide a new web UI for data import. The new UI provides better user experience and makes data import more efficient.
+
+    Using the new UI, you can preview the data to be imported, view the import process, and manage all import tasks easily.
+
+**API changes**
+
+* The TiDB Cloud API (beta) is now available to all users.
+
+    You can start using the API by creating an API key on the TiDB Cloud console. For more information, refer to [API Key documentation](/tidb-cloud/api-overview.md).
```
Comment (ran-huang) [inline tidb-cloud/release-notes-2022.md:31]:
```suggestion
    You can start using the API by creating an API key on the TiDB Cloud console. For more information, refer to [API documentation](/tidb-cloud/api-overview.md).
```

**Review by ran-huang — APPROVED** [review]
rest LGTM

**Review by qiancai — APPROVED** [review]

**Pull request merged**